Hymns/Display/DisplayHymnBottomBarViewModel.swift

```diff
@@ -42,6 +44,14 @@ class DisplayHymnBottomBarViewModel: ObservableObject {
                         return
                     }
 
+                    let title: Title
+                    if self.identifier.hymnType == .classic {
+                        title = "Hymn \(self.identifier.hymnNumber)"
+                    } else {
+                        title = hymn.title.replacingOccurrences(of: "Hymn: ", with: "")
+                    }
+                    self.title = title
+
```

Comment:
Don't think you'll need any of this once you remove the code where you pass in the title

Reply:
Removed in 783aae0